installer/windows/Product.wxs

```diff
@@ -98,7 +98,7 @@
     <Property Id="ARPPRODUCTICON" Value="storj.ico" />
     <WixVariable Id="WixUILicenseRtf" Value="agpl-3.0.rtf" />
     <WixVariable Id="WixUIBannerBmp" Value="banner.bmp" />
-    <WixVariable Id="WixUIDialogBmp" Value="dialog.bmp" />
+    <WixVariable Id="WixUIDialogBmp" Value="dialog.png" />
```

Comment:
nit - should the ID of this change since it is no longer a bmp?

Reply:
No, because this is a built-in variable of wix :)

Reply:
Oh, I didn't know png files work too!